src/js/components/put-away/PutAwaySecondPage.jsx

```diff
@@ -358,11 +362,64 @@ class PutAwaySecondPage extends Component {
     return apiClient.post(url, flattenRequest(this.state.putAway), { responseType: 'blob' })
       .then((response) => {
         fileDownload(response.data, `PutawayReport${putawayNumber ? `-${putawayNumber}` : ''}.pdf`, 'application/pdf');
+        this.fetchItems(this.state.sortBy);
         this.props.hideSpinner();
       })
       .catch(() => this.props.hideSpinner());
   }
 
+  /**
+   * Sort putaway items in order chosen by the user
+   * @public
+   */
+
+  sortByBins() {
```

Comment:
I think this should be named sortPutawayItems, because it's not sorting only by bins